docs/content/4.sdk/2.getting-started/4.logger.md

```diff
@@ -0,0 +1,169 @@
+  return defineSdkConfig(({ buildModule, config, getRequestHeaders, middlewareModule }) => ({ // [!code --]
+  return defineSdkConfig(({ buildModule, config, getRequestHeaders, loggerModule, middlewareModule }) => ({ // [!code ++]
+    logger: buildModule(loggerModule), // [!code ++]
+    // ...
+  }));
+}
+```
+
+#tab-2
+
+```ts
+// apps/storefront-unified-nuxt/sdk/config.ts
+
+// ...
+export default defineSdkConfig(({ buildModule, middlewareModule, getRequestHeaders, config }) => ({ // [!code --]
+export default defineSdkConfig(({ buildModule, loggerModule, middlewareModule, getRequestHeaders, config }) => ({ // [!code ++]
+  logger: buildModule(loggerModule), // [!code ++]
+  // ...
+}));
+```
+::
+
+
+## Configuration
+
+The default configuration is already provided, but you can customize it by providing the following options:
+
+* `level` - the minimum level of the message to be logged. The default value is `info`. Possible values are:
+  * `emergency` 
+  * `alert`
+  * `critical`
+  * `error`
+  * `warning`
+  * `notice`
+  * `info`
+  * `debug`
+* `includeStackTrace` - a boolean value that determines if the stack trace should be included in the log message. The default value is `true`.
+* `handler` - a custom handler that will be called when the message is logged.
+
+To provide the configuration for the Logger, you need to update the SDK config with the Logger module configuration by providing an object with options to the `buildModule` function, e.g.:
+```ts
+  // ...
+  logger: buildModule(loggerModule, {
+    level: 'debug',
+    includeStackTrace: false,
+    handler: myCustomHandler,
+  }),
+```
+
+### Custom handler
+There might a certain use case where you want to customize the way the Logger logs messages e.g. you want to change a log provider to a different one.
+Then you can provide your own custom handler for the Logger. The handler must implement the `LoggerInterface` interface provided in the section [types](#type).
+
+::warning
+Keep in mind that if you provide a custom handler, it will override the built-in logging functions and the other options passed to the Logger configuration will be ignored.
+::
+
+## Usage
+
+You can use the Logger in the same way as you would use any other SDK modules:
+
+::tabs{:titles='["Next.js", "Nuxt"]' class="mt-8"}
+
+#tab-1
+
+::code-group
+
+```tsx [Server Side]
+import { getSdk } from '@/sdk';
+
+function ServerComponent() {
+  const sdk = getSdk();
+
+  sdk.logger.info('Example server side log');
+
+  // ...
+}
+```
+
+```tsx [Client Side]
+import { useSdk } from '@/sdk/alokai-context';
+
+function ClientComponent() {
+  const sdk = useSdk();
+
+  useEffect(() => {
+    sdk.logger.info('Example client side log');
+  }, []);
+  
+  // ...
+}
+```
+
+::
+
+#tab-2
+
+```vue
+// component
+<script setup>
+useSdk().logger.info('Example log');
+<script>
+```
+::
+
+Now instead of using `console.log` you can use the Logger to log messages at different levels.
+
+## Type
```

Comment:
I think parts of this section should be placed in more meaningful places. E.g. LogLevel where you tell about log levels, logger interface where you tell about custom handler or just available methods of logger and so on.